docs/security.rst

```diff
@@ -60,3 +60,70 @@ Girder returns an error when a Cross-Origin request is made (one with the
 ``Origin`` header) that does not match the system configuration settings.
 Although most modern web browsers also enforce this, some additional security
 is added by enforcing it at the request level.
+
+
+Database Injection Attacks
+--------------------------
+
+Girder defends against database injection attacks by using PyMongo as the only
+pathway between the application server and the database server. PyMongo--like
+most MongoDB client libraries build up a BSON object before sending the query
+to the database. This prevents most injection attacks. See the
+`MongoDB Documentation
+<http://docs.mongodb.org/manual/faq/developers/#how-does-mongodb-address-sql-or-query-injection>`_
+for more information.
+
+Additionally, we strongly recommend running MongoDB with JavaScript disabled
```

Comment:
Change to: "...strongly recommend configuring your MongoDB server with JavaScript..." to make clear that this is not an option within Girder, but rather must be done by the server admin.